packages/react-broadcast/src/components/test-context-provider/test-context-provider.tsx

```diff
@@ -0,0 +1,52 @@
+import type {
+  TFlags,
+  TReconfigureAdapter,
+  TAdapterStatus,
+  TAdapterIdentifiers,
+} from '@flopflip/types';
+
+import React from 'react';
+import { AdapterConfigurationStatus, AdapterSubscriptionStatus } from '@flopflip/types';
+import { createAdapterContext, AdapterContext } from '@flopflip/react';
+import { createIntialFlagsContext, FlagsContext } from '../flags-context';
+
+type TProps = {
+  children: React.ReactNode;
+  flags: TFlags;
+  adapterIdentifiers?: TAdapterIdentifiers & string[];
+  reconfigure?: TReconfigureAdapter;
+  status?: TAdapterStatus;
+};
+
+const defaultProps: Pick<TProps, 'recondigure' | 'status'> = {
+  adapterIdentifiers: ['test'],
```

Comment:
What's the `test` adapter?

Reply:
Adapters have an id. This id is used to update only a respective adapter state when you use muliple. As such flag state is `Record<TAdapterIdentifier, TFlags>`. This just make clear that it's test flag state.

Reply:
Ok so in general other identifiers could be `memory`, `http`, `launchdarkly`, etc. Did I get this right?

So then my next question would be: what would be the use case for the test provider to have any other adapter other than `test`? 🤔